core/ts/Poll.ts

```diff
@@ -181,33 +175,22 @@ export class Poll implements IPoll {
    * @param index - The index of the message to process.
```

Comment:
Should update the param comments now that we pass `message` & `encPubKey`

Reply:
done